tests/load/locustfiles/locustfile.py

```diff
@@ -79,14 +80,15 @@ def __init__(self, environment) -> None:
         self.ws_greenlet: Greenlet | None = None
 
     def wait_time(self):
+        """Return the autopush wait time."""
         return self.environment.autopush_wait_time(self)
 
     def on_start(self) -> Any:
-        """Called when a User starts running."""
+        """Call when a websocket starts running."""
```

Comment:
Is 'Callback' maybe a better word for the nature of these lifecycle methods?

Reply:
I had actually tried that, but pydocstyle didn't like it since it's not in imperative mood. Now the linter is bossing us around 😆 

`D401: First line should be in imperative mood; try rephrasing (found 'Callback')`

Reply:
`D451: Developer muttered about "burn it all to the ground" which hurt linters feelings; try therapy.`